x/wasm/ibc.go

```diff
@@ -46,11 +47,11 @@ func (i IBCHandler) OnChanOpenInit(
 ) (string, error) {
 	// ensure port, version, capability
 	if err := ValidateChannelParams(channelID); err != nil {
-		return "", err
+		return version, err
```

Comment:
think that this is an upstream change related to ibc-go v6.  

Or are you saying, that since we are doing wasm contracts, we should always just return that empty string?

Yeah, ok so, I'm reading this -- I think that these are changes in ibc, related to module upgradeability.  When I get home I can give you more info, and thanks so much for reading through this code.  Really appreciate it. 

PS: I do think this is correct but not certain.